test/markdownItPlugins/pre-imported/pre-imported.md

```diff
@@ -0,0 +1,3 @@
+# Heading
```

Comment:
This is an identical copy of an existing test file, right?

Reply:
Yes, it is.  The structure of the test cases appeared to be a directory with the test case name and the supporting config file and some markdown to run it across.  I took the existing `markdownItPlugins/function` case and duplicated it for this new case.

If you have a better method or would rather the content of this test markdown reflect something else, I'm cool with that.

Reply:
What you have is exactly what I want!